src/taipy/core/config/config.schema.json

```diff
@@ -8,7 +8,9 @@
     "CORE": {
       "description": "Data nodes",
       "type": "object",
-      "properties": {},
+      "properties": {
+        "version": "3.0.0"
```

Comment:
should we have a description and a type instead of a value ?

Reply:
do you mean something like: 
```suggestion
        {"version": 
                {"description": "The version of Taipy Core",
                "type": "string"
                }
        }
```

Reply:
Yes, but @FredLL-Avaiga should know better.

Reply:
yes that would do.
What about the sequences description ?